src/app/features/vendor/vendor-listing/vendor-listing.component.ts

```diff
@@ -27,27 +28,31 @@ import { Router } from '@angular/router';
 export class VendorListingComponent implements OnInit {
   vendors!: IVendor[];
 
-  constructor(private vendorService: VendorService, private router : Router) {}
+  constructor(
+    private vendorService: VendorService,
+    private router: Router,
+    private loadingService: LoadingService
+  ) {}
 
   ngOnInit() {
+    this.loadingService.showLoader();
     this.fetchVendors();
   }
 
-/**
- * Method to get the list of vendors 
- */
-fetchVendors(): void {
-   this.vendorService.getVendors().subscribe((vendors: IVendor[]) => {
-    this.vendors = vendors;
-    this.vendors.sort((a, b) => a.name.localeCompare(b.name));
-   });
-}
+  /**
+   * Method to get the list of vendors
+   */
+  fetchVendors(): void {
+    this.vendorService.getVendors().subscribe((vendors: IVendor[]) => {
+      this.vendors = vendors;
+      this.vendors.sort((a, b) => a.name.localeCompare(b.name));
+    });
+  }
 
   /**
-   * Method to get the list of vendors 
+   * Method to get the list of vendors
    */
   navigateToView(event: IVendor): void {
-   this.router.navigate(['/vendor/view/'+event.id]);
+    this.router.navigate(["/vendor/view/" + event.id]);
```

Comment:
use single quotes. update in every ts file.

Reply:
ok